geniza/corpus/templates/corpus/snippets/document_result.html

```diff
@@ -107,7 +107,7 @@ <h3 class="sr-only">{% translate 'Tags' %}</h3>
                 {% for image in document.iiif_images|slice:":3" %}
                     <li class="image-{{ forloop.counter }}">
                         <a href="{% url 'corpus:document' document.pgpid %}">
-                            <img src="{{ image|iiif_image:"size:width=250" }}" loading="lazy">
+                            <img src="{{ image.0|iiif_image:"size:width=250" }}" loading="lazy" alt="{{ image.1 }}">
```

Comment:
maybe a comment here also to make clear it's a list of tuples of iiif image and label

Reply:
Done in 4ae76ee :+1: